bottom/connection.py

```diff
@@ -18,8 +18,8 @@ def connect(self, loop=None):
         self.reader, self.writer = yield from asyncio.open_connection(
             self.host, self.port, ssl=self.ssl, loop=loop)
         self._connected = True
-        yield from self.events.trigger(
-            "CLIENT_CONNECT", host=self.host, port=self.port)
+        asyncio.Task(self.events.trigger(
```

Comment:
The [python docs](https://docs.python.org/3/library/asyncio-task.html#task) suggest not instantiating Task directly:

> Don’t directly create Task instances: use the ensure_future() function or the BaseEventLoop.create_task() method.

From [ensure_future](https://docs.python.org/3/library/asyncio-task.html#asyncio.ensure_future) it looks like a minimal change to this:

```python
asyncio.ensure_future(
    self.events.trigger("CLIENT_DISCONNECT", host=self.host, port=self.port),
    loop=<LOOP_GOES_HERE>)
```

And in writing this comment, I realized that none of this code uses a specific loop - `events.trigger` uses the [default loop](https://github.com/numberoverzero/bottom/blob/4293d2726d2a7222faa55ca509871ee03f4e66e1/bottom/event.py#L39).  This is all documented in the open issue #13 which I haven't worked on.

---

So this is non-blocking, and the change can go in.  This will just be another area to take care of when refactoring to use a specific loop.